lib/components/submit-button.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { useSheet } from '../jss'
+
+const styles = {
+}
+
+const SubmitButton = React.createClass({
+  propTypes: {
+    isAgreed: React.PropTypes.bool.isRequired,
+    onSubmit: React.PropTypes.func.isRequired
+  },
+  componentDidMount() {
+    componentHandler.upgradeDom()
+  },
+  _onSubmit() {
+    this.props.onSubmit()
+  },
+  render() {
+    return (
+      <button onClick={this._onSubmit} type="button" className="mdl-button mdl-js-button mdl-button--raised mdl-button--colored" disabled={this.props.isAgreed ? false : true}>
```

Comment:
@watilde Hmm, do you want to use `addEventListener` ?  Like this:

```javascript
componentDidMount() {
  buttonEl.addEventListener('click', this._onSubmit)
  componentHandler.upgradeDom()
},
componentWillUnmount() {
  buttonEl.removeEventListener('click', this._onSubmit)
}
```

Is this way common practice in React?